test/e2e/common/empty_dir.go

```diff
@@ -205,6 +206,76 @@ var _ = Describe("[sig-storage] EmptyDir volumes", func() {
 	framework.ConformanceIt("should support (non-root,0777,default) [LinuxOnly] [NodeConformance]", func() {
 		doTest0777(f, testImageNonRootUid, v1.StorageMediumDefault)
 	})
+
+	It("pod should support shared volumes between containers", func() {
+		var (
+			volumeName             = "shared-data"
+			nginxVolumeMountPath   = "/usr/share/nginx/html"
+			busyBoxVolumeMountPath = "/pod-data"
+			nginxVolumeFilePath    = fmt.Sprintf("%s/index.html", nginxVolumeMountPath)
+			message                = "Hello from the busy-box container"
+			nginxContainerName     = "nginx-container"
+			busyBoxContainerName   = "busybox-container"
+			resultString           = ""
+		)
+
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "pod-sharedvolume-" + string(uuid.NewUUID()),
+			},
+			Spec: v1.PodSpec{
+				Volumes: []v1.Volume{
+					{
+						Name: volumeName,
+						VolumeSource: v1.VolumeSource{
+							EmptyDir: new(v1.EmptyDirVolumeSource),
+						},
+					},
+				},
+				Containers: []v1.Container{
+					{
+						Name:  nginxContainerName,
+						Image: imageutils.GetE2EImage(imageutils.Nginx),
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:      volumeName,
+								MountPath: nginxVolumeMountPath,
+							},
+						},
+					},
+					{
+						Name:    busyBoxContainerName,
+						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+						Command: []string{"/bin/sh"},
+						Args:    []string{"-c", "echo " + message + " > /pod-data/index.html"},
```

Comment:
And replace the file name here with an variable

Reply:
yes @msau42 , Done.